conf/prepush.sh

```diff
@@ -55,6 +62,8 @@ prepush() {
     printf "${blue}-------------------------------------------------------------${end}"
     yarn build:prod:es || exit 1
 
+    check_uncommitted_files || exit 1
```

Comment:
Should this check happen at the very beginning, instead? Or is the concern that `build:prod:es` could generate some uncommitted files?

Reply:
yeah after build:prod:es since thats what generates the variables